src/Log.cpp

```diff
@@ -45,4 +48,15 @@ LoggerStream Logger::createLogMessage(const std::string& file, const std::string
     };
     return LoggerStream(*this, source);
 }
+
+void Logger::setDebugMode(bool debug)
+{
+    _debug = debug;
+}
+
+bool Logger::isEnabled()
```

Comment:
Please start using C++ features like const and noexcept where it is appropriate like exactly here

Reply:
Okay